packages/atomic/src/pages/examples/ipx.html

```diff
@@ -26,18 +26,29 @@
 
   <body>
     <atomic-search-interface fields-to-include='["snrating","sncost"]'>
-      <atomic-ipx-layout>
-        <atomic-layout-section section="search">
-          <atomic-search-box></atomic-search-box>
-          <atomic-ipx-refine-toggle></atomic-ipx-refine-toggle>
-          <atomic-ipx-tabs>
-            <atomic-ipx-tab label="All" expression="" active></atomic-ipx-tab>
-            <atomic-ipx-tab label="HTML" expression="@filetype==html"></atomic-ipx-tab>
-            <atomic-ipx-tab label="PDF" expression="@filetype==pdf"></atomic-ipx-tab>
-            <atomic-ipx-tab label="Spreadsheet" expression="@filetype==xls"></atomic-ipx-tab>
-            <atomic-ipx-tab label="Word" expression="@filetype==doc"></atomic-ipx-tab>
-          </atomic-ipx-tabs>
-        </atomic-layout-section>
+      <atomic-ipx-modal>
+        <div slot="header">
+          <atomic-layout-section
+            style="
+              width: 100%;
+              box-sizing: border-box;
+              display: grid;
+              grid-template-columns: 1fr auto;
+              grid-gap: 0.5rem;
+            "
+            section="search"
+          >
+            <atomic-search-box></atomic-search-box>
+            <atomic-ipx-refine-toggle></atomic-ipx-refine-toggle>
+            <atomic-ipx-tabs>
+              <atomic-ipx-tab label="All" expression="" active></atomic-ipx-tab>
+              <atomic-ipx-tab label="HTML" expression="@filetype==html"></atomic-ipx-tab>
+              <atomic-ipx-tab label="PDF" expression="@filetype==pdf"></atomic-ipx-tab>
+              <atomic-ipx-tab label="Spreadsheet" expression="@filetype==xls"></atomic-ipx-tab>
+              <atomic-ipx-tab label="Word" expression="@filetype==doc"></atomic-ipx-tab>
+            </atomic-ipx-tabs>
+          </atomic-layout-section>
+        </div>
         <atomic-layout-section section="facets">
```

Comment:
Perhaps there should be a "facets" slot in the modal.

Reply:
Facets are displayed on top of the modal (with same size. See images). It is basically another modal.

![image](https://user-images.githubusercontent.com/67848363/207917851-404d5ed5-346a-4850-8b62-1f4f111f446d.png)

Having a slot for facets makes it difficult to achieve.